python/sglang/srt/layers/moe/fused_moe_triton/layer.py

```diff
@@ -823,6 +823,17 @@ def forward(self, hidden_states: torch.Tensor, topk_output: TopKOutput):
             elif TopKOutputChecker.format_is_triton_kernel(topk_output):
                 raise NotImplementedError()
 
+        if _use_aiter:
+            # NOTE: Aiter's fused_moe does not support '-1' value which is broadly used to represent the invalid expert id.
+            # If '-1' is given to gpu kernel, it may cause memory access fault.
+            if TopKOutputChecker.format_is_standard(topk_output):
+                new_topk_ids = torch.where(
+                    topk_output.topk_ids == -1,
+                    self.num_local_experts,
+                    topk_output.topk_ids,
+                )
+                topk_output = topk_output._replace(topk_ids=new_topk_ids)
+
```

Comment:
Better move `if _use_aiter:` to `if TopKOutputChecker.format_is_standard(topk_output):` clause above (#L819)
Please confirm that `self.num_local_experts` will be the same size as `topk_output.topk_ids`